pusher/pusher_client.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (
+    print_function,
+    unicode_literals,
+    absolute_import,
+    division)
+
+from pusher.util import (
+    ensure_text,
+    validate_channel,
+    validate_socket_id,
+    join_attributes,
+    data_to_string)
+
+from pusher.client import Client
+from pusher.http import GET, POST, Request, request_method
+
+import collections
+import hashlib
+import os
+import re
+import six
+import time
```

Comment:
ahh yeah, that's definitely the standard order. This was an oversight on my part – will update!